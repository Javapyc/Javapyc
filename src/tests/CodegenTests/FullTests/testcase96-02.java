class Main {
    public static void main(String[] args){
	int a = 2*2*2*2*2*2*2*2*2*2*2*2*2*2*2*2*2*2*2*2*2*2*2*2*2;
	int b = 9*9*9*9*9*9*9*9*9*0*9*9*9*9*9*9*9*9*9;
	int c = b/a;
	System.out.println(a);
	System.out.println(b);
	System.out.println(c);
    }
}