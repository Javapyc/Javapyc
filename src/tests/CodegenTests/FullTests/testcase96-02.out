33554432
0
0
